2018-03-26 11:49:11,543 [Thread-32:4656] [django.request:118] [exception:handle_uncaught_exception] [ERROR]- Internal Server Error: /learnlogs/entry/134
Traceback (most recent call last):
  File "E:\zongqiqi\biandaima\MyPlace\venv\lib\site-packages\django\template\base.py", line 475, in parse
    compile_func = self.tags[command]
KeyError: 'static'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\zongqiqi\biandaima\MyPlace\venv\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "E:\zongqiqi\biandaima\MyPlace\venv\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "E:\zongqiqi\biandaima\MyPlace\venv\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "E:\zongqiqi\biandaima\MyPlace\MySite\learninglog\learnlogs\views.py", line 88, in entry
    return render(request,"learnlogs/entry.html",context)
  File "E:\zongqiqi\biandaima\MyPlace\venv\lib\site-packages\django\shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "E:\zongqiqi\biandaima\MyPlace\venv\lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "E:\zongqiqi\biandaima\MyPlace\venv\lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "E:\zongqiqi\biandaima\MyPlace\venv\lib\site-packages\django\template\backends\django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "E:\zongqiqi\biandaima\MyPlace\venv\lib\site-packages\django\template\engine.py", line 144, in get_template
    template, origin = self.find_template(template_name)
  File "E:\zongqiqi\biandaima\MyPlace\venv\lib\site-packages\django\template\engine.py", line 126, in find_template
    template = loader.get_template(name, skip=skip)
  File "E:\zongqiqi\biandaima\MyPlace\venv\lib\site-packages\django\template\loaders\base.py", line 30, in get_template
    contents, origin, origin.template_name, self.engine,
  File "E:\zongqiqi\biandaima\MyPlace\venv\lib\site-packages\django\template\base.py", line 160, in __init__
    self.nodelist = self.compile_nodelist()
  File "E:\zongqiqi\biandaima\MyPlace\venv\lib\site-packages\django\template\base.py", line 198, in compile_nodelist
    return parser.parse()
  File "E:\zongqiqi\biandaima\MyPlace\venv\lib\site-packages\django\template\base.py", line 483, in parse
    raise self.error(token, e)
  File "E:\zongqiqi\biandaima\MyPlace\venv\lib\site-packages\django\template\base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "E:\zongqiqi\biandaima\MyPlace\venv\lib\site-packages\django\template\loader_tags.py", line 290, in do_extends
    nodelist = parser.parse()
  File "E:\zongqiqi\biandaima\MyPlace\venv\lib\site-packages\django\template\base.py", line 483, in parse
    raise self.error(token, e)
  File "E:\zongqiqi\biandaima\MyPlace\venv\lib\site-packages\django\template\base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "E:\zongqiqi\biandaima\MyPlace\venv\lib\site-packages\django\template\loader_tags.py", line 233, in do_block
    nodelist = parser.parse(('endblock',))
  File "E:\zongqiqi\biandaima\MyPlace\venv\lib\site-packages\django\template\base.py", line 477, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "E:\zongqiqi\biandaima\MyPlace\venv\lib\site-packages\django\template\base.py", line 533, in invalid_block_tag
    get_text_list(["'%s'" % p for p in parse_until], 'or'),
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 5: 'static', expected 'endblock'. Did you forget to register or load this tag?
